src/components/FilmList/Films.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import './Films.scss';
+import { isEmptyObject, isNonEmptyArray } from './deps';
+import { ListElement } from './type';
+
+class Films extends React.Component<ListElement> {

```

Comment:
This declaration can be improved by adding the type of the state.
`
interface State { isShow: boolean; }

class Films extends React.Component<ListElement, State> { ... }
`